src/main/java/nextstep/ladder/domain/line/LadderLine.java

```diff
@@ -0,0 +1,58 @@
+package nextstep.ladder.domain.line;
+
+import nextstep.ladder.generator.LadderPointGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LadderLine {
+    private final List<Point> points;
+
+    private LadderLine(List<Point> points) {
+        this.points = points;
+    }
+
+    public int move(int position) {
+        return points.get(position).move();
+    }
+
+    public static LadderLine init(int sizeOfPerson, LadderPointGenerator ladderPointGenerator) {
+        List<Point> points = new ArrayList<>();
+        Point point = initFirst(points, ladderPointGenerator);
+        point = initBody(sizeOfPerson, points, point, ladderPointGenerator);
+        initLast(points, point);
+
+        return new LadderLine(points);
+    }
+
+    private static Point initBody(int sizeOfPerson, List<Point> points, Point point, LadderPointGenerator ladderPointGenerator) {
+        for (int i = 1; i < sizeOfPerson - 1; i++) {
+            Point newPoint = point.next(ladderPointGenerator);
+            points.add(newPoint);
+            point = newPoint;
+        }
+        return point;
```

Comment:
point = newPoint를 하면 결국 파라미터로 받은 point의 값을 변경하게 되는건 동일하겠네요
사실 크게  차이는 없는데, 저는 파라미터로 받은 값은 불변한게 좋다고 생각하는 편이어서요.. (습관처럼  파라미터  값을 변경해 사용하다보면 실수를 할수도 있을  것 같아서요)

```suggestion
        Point newPoint = point
        for (int i = 1; i < sizeOfPerson - 1; i++) {
            newPoint = newPoint.next(ladderPointGenerator);
            points.add(newPoint);
        }
        return newPoint;
```

제가 의도한  내용은 위와 같습니다~